cts/cts-scheduler.in

```diff
@@ -389,6 +403,53 @@ function test_results {
     fi
 }
 
+
+if test -z "$single_test" && test "$num_jobs" -ne 1 \
+        && command -v parallel >/dev/null 2>&1; then
+    _ACC_FILE=()
+    _ACC_DESC=()
+    _ACC_ARGS=()
+    export -f do_test_inner error failed info normalize show_test
+    export base create_mode diff_opts failed io_dir test_cmd
+    export CRM_EX_OK CRM_EX_ERROR CRM_EX_NOT_INSTALLED \
+           CRM_EX_USAGE CRM_EX_NOINPUT
+    do_test() {
+        test "$num_jobs" -ne 0 || num_jobs=100%
+        test $# -gt 1 || test $# -lt 1 || test "$1" != "EOT" || {
+            parallel -j "$num_jobs" --halt soon,fail=100 --plus \
+              --tagstring '{= $_="\033[30;3".(++$::color%8)."m" =}' \
```

Comment:
I don't take it too seriously, knowing that both Fedora and FSF (!) was
OK with that.

Reply:
Author's post on the topic:
https://lists.gnu.org/archive/html/parallel/2013-11/msg00006.html

The best good will (we have nothing to do with academia nor with journal
articles etc.) action on our side is to put a "citation" remark as
a sort of a footnote in `INSTALL.md`, IMHO.  Plus I don't think we
shall play any tricks on behalf of the users -- we never now if they
don't happen to be writing a scientific article using pacemaker test
suite as a data set :-)

There are other implementations for the same interface, e.g. the Rust
one (https://github.com/mmstick/parallel) which doesn't appear very
maintained (and doesn't support some features of GNU version I used).

Reply:
Also https://www.gnu.org/licenses/gpl-faq.en.html#RequireCitation
Looks like a gray area, but not putting off much, since you check
that once and you are done with it.  Ditto in CI, extra command
in a preamble and done (unless e.g. Ubuntu/Debian goes the explicit
disarming route -- even though, Travis and likely kronosnet CI are
currently not multi-core, IIRC, so the benefits are questionable
there.